app/javascript/utilities/color/accentCalculator.js

```diff
@@ -7,6 +7,11 @@
  */
 export function brightness(color, amount = 1) {
   const rgbObj = hexToRgb(color);
+
+  if (!rgbObj) {
```

Comment:
Reading Notes: this guard clause is because later on you're expecting a hash.  But we may have a false/null object from our hexToRgb, which occurs if the given value does not match the regular expression for a hex color.

https://github.com/forem/forem/blob/aa14208d609a233c578db93ad5844e3a287d69e8/app/javascript/utilities/color/accentCalculator.js#L28-L37

So this is a good guard clause.  Our other option would be to modify `hexToRgb` to either return an "empty" object (e.g. an Object who's keys are an empty array) or to throw an exception and catch that here.

Reply:
Yep, exactly. I agree an empty object would be an elegant solution too, but I chose the route that required less refactoring 😄 

Another bit of context is that the code already in this file is based largely on a Ruby library we use in the backend, so I figure the less we deviate from that, the better.